cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -70,6 +68,11 @@ message DisplayVariable {
   //     minCpu: 2
   //     minRamGb: 6
   GooglePropertyExtension x_google_property = 14;
+
+  // Text describing the validation rules for the property. Typically shown
+	// after an invalid input.
+	// Optional. UTF-8 text. No markup. At most 128 characters.
+	string validation = 15;
```

Comment:
Guessing these should be spaces instead of tabs?

```suggestion
  string validation = 15;
```

Reply:
This seems like a Github bug. It shows up fine locally for me. Let me repost and see if that fixes it.